src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,130 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .m-dropdown {
+    position: relative;
+    display: inline-block;
+
+    .m-dropdown__menu {
+      position:absolute;
+      left: 0;
+      min-width: 100%;
+    }
+  }
+
+  .m-dropdown__trigger {
+    background-image: url('./icon-chevron-thin-down.svg');
+    background-position: right 0.5rem center;
+    background-repeat: no-repeat;
+    background-size: 0.75rem 0.75rem;
+    padding-right: 2rem;
+
+    &.-active {
+      background-image: url('./icon-chevron-thin-up.svg');
+    }
+  }
+
+  .m-dropdown__menu {
+    background-color: set-color(white);
+    border: 0.0625rem solid set-color(grey, 30);
+    border-radius: $border-radius;
+    box-shadow: 0 1px 4px 0 rgba(set-color(black), 0.1);
+    font-size: $text;
+    z-index: 1;
+    margin: 0;
+    padding: 0.5rem 0;
+    display: none;
+
+    &.-active {
+      display: block;
+    }
+
+    & > .-divider {
+      border-bottom: 0.0625rem solid set-color(grey, 20);
+      margin: 0.5rem 0;
+    }
+
+    & > a.m-dropdown__menu-item {
+      color: inherit;
+
+      &:hover {
+        text-decoration: inherit;
+      }
+
+      &:hover:not(.-disabled):not(.-active),
+      &.-hover:not(.-disabled):not(.-active),
+      &:focus:not(.-disabled):not(.-active),
+      &.focus:not(.-disabled):not(.-active)
+      {
+        background-color: set-color(grey, 20);
+      }
+    }
+
+    & > .m-dropdown__menu-item {
+
+      height: 2rem;
+      padding-left: 1rem;
+      padding-right: 1rem;
+      display: flex; // Ver clase -flex-ellipsis
+      align-items: center;
+      white-space: nowrap;
+
+      &.-h--auto {
```

Comment:
@mattnickles is it necessary this class/ utility for dropdowns?

Reply:
Yes, let's leave for now. We'll create a global sizing utility in a future release.